pqarrow/writer/writer.go

```diff
@@ -239,3 +239,62 @@ func (w *float64ValueWriter) WritePage(p parquet.Page) error {
 
 	return nil
 }
+
+type booleanValueWriter struct {
+	b   *array.BooleanBuilder
+	buf []bool
+}
+
+func NewBooleanValueWriter(b builder.ColumnBuilder, numValues int) ValueWriter {
+	res := &booleanValueWriter{
+		b: b.(*array.BooleanBuilder),
```

Comment:
I think it would be nice to write an `OptBoolBuilder` and use that for efficient parquet to arrow conversion. Can be added in the future but I think it's worth exploring now since I don't think it should be too much code. See `optbuilders.go` and especially `OptInt64Builder`

Reply:
Done